webrtc.html

```diff
@@ -2354,11 +2356,126 @@
           SDP this candidate is associated with.</dd>
+
+          <dt>DOMString foundation</dt>
+
+          <dd>A unique identifier that allows ICE to correlate candidates that
+          appear on multiple <code><a>RTCIceTransport</a></code>s.</dd>
+
+          <dt>unsigned long priority</dt>
+
+          <dd>The assigned priority of the candidate. This is automatically
+          populated by the browser.</dd>
+
+          <dt>DOMString ip</dt>
+
+          <dd>The IP address of the candidate.</dd>
+
+          <dt>RTCIceProtocol protocol</dt>
+
+          <dd>The protocol of the candidate (<code>udp</code>/<code>tcp</code>).</dd>
+
+          <dt>unsigned short port</dt>
+
+          <dd>The port of the candidate.</dd>
+
+          <dt>RTCIceCandidateType type</dt>
+
+          <dd>The type of candidate.</dd>
+
+          <dt>RTCIceTcpCandidateType tcpType</dt>
+
+          <dd>If <code>protocol</code> is <code>tcp</code>, <code>tcpType</code> represents
+          the type of TCP candidate. Otherwise, <code>tcpType</code> is not present in the
+          dictionary.</dd>
+
+          <dt>DOMString relatedAddress</dt>
+
+          <dd>For a candidate that is derived from another, such as a relay or reflexive
+          candidate, the <dfn>relatedAddress</dfn> is the IP address of the candidate that
+          it is derived from. For host candidates, the <code>relatedAddress</code>
+          is not present in the dictionary.</dd>
+
+          <dt>unsigned short relatedPort</dt>
+
+          <dd>For a candidate that is derived from another, such as a relay or reflexive
+          candidate, the <dfn>relatedPort</dfn> is the port of the candidate that
+          it is derived from. For host candidates, the <code>relatedPort</code>
+          is not present in the dictionary.</dd>
+          </dd>
         </dl>
         <div class="note">
           <p>The constructor on <code><a>RTCIceCandidate</a></code> exists
           for legacy compatibility reasons only.</p>
         </div>
+
+        <section>
+          <h4>The RTCIceProtocol</h4>
+
+          <p>The <dfn>RTCIceProtocol</dfn> represents the protocol of the ICE candidate.</p>
+
+          <dl class="idl" title="enum RTCIceProtocol">
+            <dt>udp</dt>
+
+            <dd>A UDP candidate, as described in [[!ICE]].</dd>
+
+            <dt>tcp</dt>
+
+            <dd>A TCP candidate, as described in [[!RFC6544]].</dd>
+          </dl>
+        </section>
+
+        <section>
+          <h4>The RTCIceTcpCandidateType</h4>
+
+          <p>The <dfn>RTCIceTcpCandidateType</dfn> represents the type of the ICE TCP
+          candidate, as described in [[!RFC6544]]. Browsers MUST gather <code>active</code>
+          TCP candidates and only <code>active</code> TCP candidates. Servers and other
+          endpoints MAY gather <code>active</code>, <code>passive</code> or <code>so</code>
+          candidates.</p>
+
```

Comment:
I'm guessing it's in JSEP or is a TODO of JSEP.  In either case, it wouldn't be appropriate in this PR since this PR is about providing more information, not about defining behavior.